src/libstore/globals.hh

```diff
@@ -1262,6 +1262,16 @@ public:
           store paths of the latest Nix release.
         )"
     };
+
+    Setting<uint64_t> largePathWarningThreshold{
+        this,
+        std::numeric_limits<uint64_t>::max(),
+        "large-path-warning-threshold",
```

Comment:
Potential to improve discovery a bit by alphabetical grouping. Puts it next to `warn-dirty`.

```suggestion
        "warn-large-path-threshold",
```